lib/ui/viewer/file/zoomable_image.dart

```diff
@@ -161,7 +163,11 @@ class _ZoomableImageState extends State<ZoomableImage>
 
     if (!_loadingFinalImage && !_loadedFinalImage) {
       _loadingFinalImage = true;
-      getFile(_photo).then((file) {
+      getFile(
```

Comment:
Assuming that we don't need to delete the original file from the cache. App will eventually do it?

Reply:
Right, during next app startup